packages/subgraph/schema.graphql

```diff
@@ -462,152 +462,356 @@ type StandardCallback @entity {
   magicNumber: Bytes!
 }
 
-# DAO
+#############################
+########### DAO #############
+#############################
```

Comment:
This pattern is only for us to see the sections better ? or it will contribute to the dev-portal in some way?

Reply:
just for us